### DISCUSSION TITLE
Print Thin Walls - No longer an Option

### DISCUSSION THREAD

**Discussion by Dwamies** [5axes/Calibration-Shapes#109] [General]
As per the title, the setting for Print Thin Walls, along with several others, has been removed completely by Ultimaker in Cura V5. This affects (at a minimum) the suggested print settings for the Tolerance Test. 

What would be a good solution to stop people like myself from spending a large volume of time searching for a setting that isn't available? Maybe a simple notation after the suggested setting like:
Print Thin Wall: False (Cura V4 only)

Sauce: https://support.ultimaker.com/hc/en-us/articles/4792349488668-What-print-settings-have-been-removed-from-Cura-5-

I have never modified anything on Github really and I presumed it would not have been welcome for me to just go ahead and start modifying the Wiki?

**Comment by 5axes** [comment]
Hello,
Thanks for your report, but As **Print Thin Walls** is still a parameter ,of V5 as well as 5.1 **NO** please don't modify anything. On top of that as the *Print Thin Walls* give access to parameters not easy to understand in V5 it's higtly recommanded to live it unchecked , in order to keep the default behaviour of Cura in most of the case (My point of View) .

Cura 5.0
![image](https://user-images.githubusercontent.com/11015345/184302764-5f8064df-cfe7-4dc9-978a-a13b47707f48.png)

Cura 5.1
![image](https://user-images.githubusercontent.com/11015345/184302837-11acb6e0-489b-426b-b6d5-6611488bb22e.png)


Now if you have the reference of real parameters, no longer available in V5.0 and mentioned in the documentation, Please don't hesitate to report them.
